typescript-test-samples/step-functions-local/README.md

```diff
@@ -0,0 +1,34 @@
+## Step Functions Local Testing using Jest
+[Step Functions Local testing with mock configuration](https://aws.amazon.com/blogs/compute/mocking-service-integrations-with-aws-step-functions-local/) 
+provides the capability to mock AWS service integrations that are present in a state machine. This helps in testing the 
+state machine in isolation.
+
+This is sample project which showcases how to run Step Functions local tests with mocks using Jest instead of running ad-hoc CLI commands. [Testcontainers](https://www.testcontainers.org/) is used to run [Step Functions Local Docker image](https://docs.aws.amazon.com/step-functions/latest/dg/sfn-local-docker.html).
+
+This is not a replacement of the strategy shown in above blog but another way to test Step Functions state machine with
+better assertion capabilities. Teams currently using Javascript / Typescript for Serverless development can leverage this strategy in their
+current applications.
+
+### Prerequisites
+ - Node.js v14 or newer
+ - [Jest](https://jestjs.io/docs/getting-started)
+ - [Testcontainers](https://www.testcontainers.org/)
+ - [Docker](https://www.docker.com/)
+
+### Running Tests
+> Make sure docker engine is running before running the tests.
+
+In order to install the dependencies, just run below command from project "src" subdirectory:
+```bash
+npm install
+```
+
+In order to run the tests, just run below command from project "src" subdirectory:
+```bash
+npm run test
```

Comment:
Getting errors running tests:

```● lead-generation › step-function-local › Check Container is running

    No Docker client strategy found```

I'm using Podman, which looks like its supposed to be supported...
https://node.testcontainers.org/supported-container-runtimes/

Any ideas?

Reply:
According to [atomicjar.com](https://www.atomicjar.com/2021/10/docker-on-windows-and-macos/), Podman is a Linux tool primarily, but it can also work on macOS and Windows. However, compatibility with Testcontainers using Podman as the underlying container runtime is not guaranteed. The article states that there were a higher number of failures observed when using Podman with Testcontainers, indicating potential compatibility issues. Therefore, it's recommended to use Docker Desktop on macOS for the best compatibility with Testcontainers.

Reply:
I've put docker as a requirement to run the test sample